driver/razermouse_driver.h

```diff
@@ -8,6 +8,7 @@
 
 #define USB_DEVICE_ID_RAZER_OROCHI_2011 0x0013
 #define USB_DEVICE_ID_RAZER_DEATHADDER_3_5G 0x0016
+#define USB_DEVICE_ID_RAZER_DEATHADDER_3_5G_BLACK 0x0029
```

Comment:
please sort this list by PID, so below `USB_DEVICE_ID_RAZER_MAMBA_2012_WIRELESS`